css/style.css

```diff
@@ -0,0 +1,232 @@
+html { 
+  background: url("../images/Skola.jpg") no-repeat center center fixed;
+  background-size: cover;
+}
+
+body {
+  margin: 0;
+}
+
+.page-container{
+  position: relative;
+  min-height: 100vh;
+}
+
+.content-wrap{
+  padding-bottom: 15px;
+}
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  text-align: end;
+  padding-top: 10px;
+  padding-right: 10px;
+}
+
+li {
+  display: inline;
+}
+
+a:hover {
+  color: #434343;
+}
+
+a {
+  color: #f5f5f5;
+  font-family: 'K2D', sans-serif;
+  text-decoration: none;
+  margin: 5px;
+}
+
+/* INDEX PAGE */
+
+.title-info {
+  margin: 60px;
+}
+
+h1 {
+  color: #f5f5f5;
+  text-align: center;
+  text-shadow: 1px 1px #000000;
+  font-family: 'K2D', sans-serif;
+  font-size: 45px;
+  margin-bottom: 0px;
+}
+
+h3 {
+  color: #f5f5f5;
+  text-align: center;
+  text-shadow: 1px 1px #000000;
+  font-family: 'K2D', sans-serif;
+  font-size: 20px;
+  margin-top: 0px;
+}
+
+.footer{
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  color: #f5f5f5;
+  background-color:#e1e1e166;
+  padding: 5px 0px;
+  text-align: center;
+}
+
+.profile_pic {
+  width: 191px;
+  height: 261px;
+  margin-bottom: 0px;
+  margin-right: 10px;
+  margin-left: 10px;
+  align-content: left;
+  display: inline;
+  float: left;
+}
+.profile_discrip {
+  display: inline;
+  margin: 1%;
+}
+
+.personal {
+  display: flex;
+  margin-left: 20%;
+  margin-right: 20%;
+  margin-top: 5%;
+  background-color: #e1e1e166;
+  padding: 3%;
+  border-radius: 4%;
+  font-weight: bold;
+  color: #2b344a;
+  font-family: 'K2D', sans-serif;
+  font-size: 14px;
+}
+
+/* CONTACT PAGE */
+
+.form {
+  text-align: left;
+  margin-left: 20%;
+  font-weight: bold;
+  color: #2b344a;
+  font-family: 'K2D', sans-serif;
+  margin-top: 5px;
+}
+
+.form-input {
+  font: unset;
+  width: 70%;
+  padding: 5px;
+  margin-top: 7px;
+  background-color: #e1e1e166;
+  color: #2b344a;
+  font-family: 'K2D', sans-serif;
+  font-size: 13px;
+}
+
+.form-submit {
+  background-color: #e1e1e166;
+  color: #2b344a;
+  font-family: unset;
```

Comment:
I don't think you need to do the `unset`.

Reply:
Correct, it has been removed.